NEWS.md

```diff
@@ -224,7 +224,7 @@ and multiple response variables.
 # crunch 1.18.4
 * All catalog objects now have an `as.data.frame()` method.
 * The list of dataset "weight variables" can now be set with `modifyWeightVariables()`, `weightVariables(ds) <- ds$newWeight` or `is.weightVariables(ds$var) <- TRUE`
-* Users with account admin privileges can now `expropriateUser()` to transfer datasets, projects, and other objects owned by one user to another, as when that user has left your organization.
```

Comment:
editing past news isn't a great way to communicate this change. It should receive a new entry at the top of the file (current version) so we can inform users that the function has changed names.

The few users who were aware of and used this function may be confused by its sudden disappearance; we might even want to keep the old function name and have it throw an error to inform them.